game_logic.h

```diff
@@ -0,0 +1,16 @@
+#include <iostream>
+#include <string>
+#include <cstdlib> 
+#include <ctime>
+#include <vector>
+#include <unordered_map>
+#include <utility>
+#include "board.h"
+
+std::string getPlayerInput(Board &board, Player player, int askCoordinate);

```

Comment:
#should define an `enum class` for `askCoordinate`

#could rename to `askAxis`, which is more accurate